pkg/cover/cover.go

```diff
@@ -48,3 +60,111 @@ func (cov Cover) Serialize() []uint32 {
 	}
 	return res
 }
+
+const EXT string = ".ko"
```

Comment:
This should not be an exported name. Identifiers starting with capital letters are exported from packages.